ports/sqlgen/vcpkg.json

```
@@ -0,0 +1,57 @@
+{
+  "name": "sqlgen",
```

```suggestion
  "name": "getml-sqlgen",
```

Searching for "sqlgen" returned two projects with similar names:

* https://github.com/si3nloong/sqlgen
* https://github.com/drone/sqlgen

The name `sqlgen` does not appear when searching repology or debian packages.